lib/circuit.js

```diff
@@ -168,12 +168,24 @@ class CircuitBreaker extends EventEmitter {
       this[STATUS] = new Status(this.options);
     }
 
-    this[STATE] = CLOSED;
```

Comment:
I think this should stay as the default, and in the `if (options.state)` block, it can be updated if needed.

Reply:
sounds good